### PR TITLE
Add problems Agile Software Development

### PR DESCRIPTION
**Pull request by Winternight9**
Add agile software development problem. It is useful agile and can be applied to project.

### CODE REVIEW

File: agile-software-development/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+5. Projects are built around motivated individuals, who should be trusted
+6. Face-to-face conversation is the best form of communication (co-location)
+7. Working software is the primary measure of progress
+8. Sustainable development, able to maintain a constant pace
+9. Continuous attention to technical excellence and good design
+10. Simplicity—the art of maximizing the amount of work not done—is essential
+11. Best architectures, requirements, and designs emerge from self-organizing teams
+12. Regularly, the team reflects on how to become more effective, and adjusts accordingly
+
+Which Manifesto for Agile Software Development to solve problem or best describe activity below.
```
Comment (jbrucker) [inline agile-software-development/README.md]: "Which statement from the manifesto would solve or best describe the ..."

File: agile-software-development/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+1. ____ Most way to reduces the cycle time typically taken when questions and answers are mediated through phone,chat, or email is ?
```
Comment (jbrucker) [inline agile-software-development/README.md]: Many punctuation errors, including some I mentioned in class.   Find and fix them.

Most -> Best
reduces -> reduce

File: agile-software-development/README.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+2. ____ Sometimes, the customers want to change some requirements that make you doing more work but you need to accept it. What Agile Software Development That you following in this activity ?
```
Comment (jbrucker) [inline agile-software-development/README.md]: grammar errors. Use an online tool to find and fix them.  Also capitalization error.

File: agile-software-development/README.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+3. ____ When developers doing work without talking with business people It can cause some problems developer can doing wrong thing because they don’t have enough feedback. What Agile Software Development that solved this problem ?
```
Comment (jbrucker) [inline agile-software-development/README.md]: More wording and capitalization errors.

File: agile-software-development/README.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+
+4. ____ Some bad development plans have long iteration around 1 months or more that make you work so bad because long iteration mean less retrospective less feedback.What Agile Software Development that solved this problem ?
```
Comment (jbrucker) [inline agile-software-development/README.md]: More grammar errors.  There are so many that I stopped checking here.

"aounrd 1 months" -> "around 1 month"     Google Docs would catch this error!

Did you use grammar checker(s) before submitting this?

Comment (Winternight9) [reply agile-software-development/README.md]: I'm too careless. I think just using google docs to check grammar is enough. Next time I will always use grammar checkers before submitting work.

**Comment by Winternight9** [comment]
I'm too careless. I think just using google docs to check grammar is enough. Next time I will always use grammar checkers before submitting work.

**Pull request merged**